src/Symfony/Component/Debug/CHANGELOG.md

```diff
@@ -1,6 +1,18 @@
 CHANGELOG
 =========
 
+3.0.0
+-----
+
+* removed classes, methods and interfaces deprecated in 2.x
+
+2.7.0
```

Comment:
you should at least remove the line about the new function in the C extension, given it is not merged yet